### ISSUE TITLE
Additional Exensions

### ISSUE THREAD

**Issue by VaclavElias** [VaclavElias/stride-code-only#16]
https://github.com/Doprez/Doprez.Stride
There is a Worldposition extension in my repo and there is another useful repo from DfKeenan
https://github.com/dfkeenan/StrideToolkit that has even more useful extensions

**Comment by IXLLEGACYIXL** [comment]
#24

**Comment by IXLLEGACYIXL** [comment]
i think that was allready done?

**Comment by VaclavElias** [comment]
Half of it. Thanks for a reminder. I need to review https://github.com/dfkeenan/StrideToolkit architecture/folder structure, as he was ahead with the toolkit, I need to make sure that our toolkit has all useful parts.

**Comment by VaclavElias** [comment]
I checked this one closely again. I think I follow the structure of this toolkit but there is more extensions we could use https://dfkeenan.github.io/StrideToolkit/api/ so maybe we can keep this open and wait till we take whatever is needed. I am going to spilt the initial description into tasks..

**Comment by dfkeenan** [comment]
Hi,

I recently got asked about updating my repo (https://github.com/dfkeenan/StrideToolkit). I have directed them here as I probably won't. 😅

Please feel free to take as much of it as you want. I think it would be better to have the one official community toolkit.

**Comment by VaclavElias** [comment]
You are right @dfkeenan. I have to review, that all was taken and documented accordingly.
